### PR TITLE
Contact form collaboration 

### PR DESCRIPTION
**Pull request by eerapheal**
Contact form completed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -177,12 +199,35 @@ <h2 class="sec-3-div-h1">About me</h2>
...
+        <form class="contact-form" action="https://formspree.io/f/mknylokq" method="post">
+            <input class="name-input" type="text" name="name" placeholder="Name" maxlength="30" required>
+            <div class="line3"></div>
+            <input class ="email-input" type="email" name="Email" placeholder="Email" required>
+            <div class="line3"></div>
+            <textarea class="msg-input" name="Message" placeholder="Enter Text Here" maxlength="500" required cols="30" rows="10"></textarea>
+            <button class="contact-btn" type="submit">Get in touch</button>
+        </form>
+      </article>
+      <div class="line4"></div>
```
Comment (muneebulrehman) [inline index.html:217]: Consider adding some margin between the heading and the form as in the figma design. Also, consider removing the outline when the name and email inputs fields are in focus.
Here is the reference of your app.
image

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @eerapheal , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Ralph,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  💯🌟 

## Highlights 👍
- Form validation properly implemented
- Form action and method properly implemented


<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by eerapheal** [comment]
Thanks for the review.

**Pull request merged**